### PR TITLE
Contacts

### PR DESCRIPTION
**Pull request by or-else**

### CODE REVIEW

**Comment by or-else** [comment]
@aforge Please take a look. Thanks!

**Comment by aforge** [comment]
It looks like renaming the app "ios -> Tinodios" breaks the project.

sc1

One thing that I found is the incorrect project path in the workspace config (I commented on it). But there's more.

**Comment by aforge** [comment]
We'll also need to rename the directory with the app code "ios -> Tinodios".
And potentially the workspace - but that's not important at the moment.

File: TinodeSDK/Topic.swift [hunk 1]
```diff
@@ -911,4 +911,35 @@ public class ComTopic<DP: Codable>: Topic<DP, PrivateType, DP, PrivateType> {
     override init(tinode: Tinode?, name: String, desc: Description<DP, PrivateType>) throws {
         try super.init(tinode: tinode, name: name, desc: desc)
     }
+
+    public var isArchived: Bool {
+        get {
+            let archived = priv?["arch"]
```
[the remark this answers is not in the document]
Comment (or-else) [reply TinodeSDK/Topic.swift]: done

File: TinodeSDK/Topic.swift [hunk 2]
```diff
@@ -911,4 +911,35 @@ public class ComTopic<DP: Codable>: Topic<DP, PrivateType, DP, PrivateType> {
...
+                return x
+            default:
+                return false
+            }
+        }
+    }
+
+    public var comment: String? {
+        get {
+            let comment = priv?["comment"]
```
[the remark this answers is not in the document]
Comment (or-else) [reply TinodeSDK/Topic.swift]: done

**Pull request merged**